### DISCUSSION TITLE
Inspector 창에서 TextMeshProUGUI 할당

### DISCUSSION THREAD

**Discussion by Asipe42** [naverz/zepeto-studio-kor#1021] [5. Q&A]
TextMeshProUGUI를 사용하고 싶은데
Insepctor 창에서 allocate가 안되요 Text를 받고 있네요.

해결 방법 있을까요?
![00000](https://user-images.githubusercontent.com/86538738/179147139-357b8187-7fc2-47b2-9f1e-05029aeed0e1.PNG)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 사용하신 월드 버전이 어떻게 되시나요?
재현이 안되고 있어서 상세 개발 환경과 함께 이슈 게시판에 올려주시면 좋을 것 같습니다. 

<img width="377" alt="스크린샷 2022-07-15 오후 1 59 53" src="https://user-images.githubusercontent.com/100118650/179154026-82dc238c-be7b-4261-a196-b89bc02f6e87.png">
 

참고로 인스펙터에 할당이 안되는 이슈는 https://github.com/naverz/zepeto-studio-kor/issues/979 로 제보된 적 있습니다.

**Reply by Asipe42** [reply]
일시적인 문제였어요. 제페토 패키지를 다시 인스톨 하니 정상 작동합니다!
